release/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+echo "Beginning"
+while [[ -n "${1-}" ]] ; do
+  case "${1}" in
+    --repository-username*)
+      SONATYPE_USERNAME=${1#*=}
+      ;;
+    --repository-password*)
+      SONATYPE_PASSWORD=${1#*=}
+      ;;
+    --signing-key*)
+      GPG_SIGNING_KEY=${1#*=}
+      ;;
+    --signing-password*)
+      GPG_SIGNING_PASSWORD=${1#*=}
+      ;;
+    *)
+      echo "Unknown option '${1}'"
+      exit 1
+      ;;
+  esac
+  shift
+done
+
+echo Building and testing...
+./gradlew build
+
+echo Publishing to Sonatype, closing repository, checking uploaded artifacts and releasing repository...
```

Comment:
I think that the changes look good so we can uncomment the release of the repository in line 32 and are good to go.